README.md

```diff
@@ -223,3 +223,6 @@ $telemetryClient->getContext()->getOperationContext()->setId(\ApplicationInsight
 $telemetryClient->trackMetric('differentOperationMetric', 11);
 $telemetryClient->flush();
 ```
+## Code of conduct
+
+This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
```

Comment:
Cool, thank you. Would you please add references for actual conduct and FAQ.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.